src/payment/strategies/klarna/klarna-credit.ts

```diff
@@ -1,8 +1,8 @@
 export default interface KlarnaCredit {
-    authorize(params: any, callback: (res: KlarnaAuthorizationResponse) => void): void;
+    authorize(params: any, data: KlarnaUpdateSessionParams, callback: (res: KlarnaAuthorizationResponse) => void): void;
```

Comment:
Do we have duplicate lines here?

Reply:
@Sure-Round It's an overloaded method, I'm following same recommendation that Catalina gave me [here](https://github.com/bigcommerce/checkout-sdk-js/pull/554#discussion_r275182376) and I consider that this is a similar case